less/styles.less

```diff
@@ -77,9 +76,23 @@
 }
 
 .featured-footer{
-  margin-top:50px;
+  margin-top: 50px;
 }
 
 .logo-eclipse-default-mobile {
-  max-width:165px;
-}
\ No newline at end of file
+  max-width: 165px;
+}
+
+@media (min-width: @screen-sm-min) {
+  #main-menu li .dropdown-menu:before {
+    display: none;
+  }
+
+  #main-menu li .dropdown-menu {
+    border-top: 5px solid @brand-tertiary;
+  }
```

Comment:
I did that because there is a visual bug:

![image](https://user-images.githubusercontent.com/333276/113599517-02434080-963f-11eb-8ef6-5acbcb0cc947.png)

As you can see, there is one pixel above the top border. The theme is using a `before` pseudo-element but I think it would be better to use a border.

Or maybe this is intended? If not, I can probably submit a pull request to fix it upstream.